server/lib/model/action.dart

```diff
@@ -45,19 +42,101 @@ class ChangeLineItemStatusAction implements Action {
       ..type = proto.Action_Type.CHANGE_LINE_ITEM_STATUS
       ..changeLineItemStatusParams = (proto.ChangeLineItemStatusParams()
         ..status =
-            proto.ChangeLineItemStatusParams_Status.valueOf(_statusValue));
+            proto.ChangeLineItemStatusParams_Status.valueOf(statusValue));
   }
+}
 
-  /// Changes the status of the line item using the DV360 client.
-  /// TODO(@thu5): Record the return status and implement run history.
-  @override
-  Future<void> run(DisplayVideo360Client client, Target target) async {
-    final lineItemTarget = target as LineItemTarget;
+/// A class that represents changing the line item bidding strategy.
+class ChangeLineItemBiddingStrategyAction implements Action {
```

Comment:
nit: As we add more and more rules, this file and proto will become super large. it is better to split them into individual file by rule.

Reply:
Ok, yes, that makes sense. I'll split them up.